info_db.py

```diff
@@ -0,0 +1,75 @@
+import logging
+import sqlite3
+from pathlib import Path
+from sqlite3 import Error
+
+Path("info_database.db")
+CREATE_TABLE_SQL = """
+CREATE TABLE IF NOT EXISTS bin_info (
+symbol TEXT,
+status TEXT,
+);
+"""
+INSERT_INTO_SQL = """
+INSERT INTO bin_info
+(symbol,status)
+VALUES(1,2);
```

Comment:
According to the examples in official docs you need to pass schema but not the number